File: bard.rb
$:.unshift(File.dirname($0))

require 'gemuo/simple'
require 'gemuo/engines/skills'
require 'gemuo/engines/stats'
#require 'gemuo/engines/use'
require 'gemuo/engines/debug'

client = GemUO::SimpleClient.new

skills = [ GemUO::SKILL_PROVOCATION,
           GemUO::SKILL_PROVOCATION,
           GemUO::SKILL_PEACEMAKING,
         ]
GemUO::Engines::StatLock.new(client, [100, 25, 100]).start
GemUO::Engines::EasySkills.new(client, skills).start
#GemUO::Engines::Use.new(client, 0xe9c).start # drums
GemUO::Engines::MessageDump.new(client).start

client.run

File: src/bard.py
#!/usr/bin/python

from uo.skills import *
from gemuo.simple import simple_run
from gemuo.engine.stats import StatLock
from gemuo.engine.training import SkillTraining
from gemuo.engine.messages import PrintMessages
from gemuo.engine.guards import Guards
from gemuo.engine.watch import Watch

skills = (
    SKILL_PROVOCATION,
    SKILL_PROVOCATION,
    SKILL_PEACEMAKING,
)

def run(client):
    PrintMessages(client)
    Guards(client)
    Watch(client)
    StatLock(client, (100, 25, 100))
    return SkillTraining(client, skills, round_robin=True)

simple_run(run)
